README.md

```diff
@@ -73,3 +73,21 @@ EOF
 ```
 kubectl create -f <rdma-app.yaml>
 ```
+
+# RDMA Shared Device Plugin with NFD (Node Feature Discovery)
+[NFD](https://github.com/kubernetes-sigs/node-feature-discovery) can be use to discovery the node capabilities, and expose them as labels to the node
```

Comment:
it should be either: "can be used for discovery" or "can be used to discover" pick one :)